python/veles/schema/model.py

```diff
@@ -246,6 +246,7 @@ def __init_subclass__(cls, **kwargs):
         if not hasattr(cls, 'object_types'):
             cls.object_types = {}
         elif cls.object_type is not None:
+            cls.object_type = six.text_type(cls.object_type)
```

Comment:
not a bad idea, but needs verification that object_type is of a sane type (since every object is str'able in python)